src/components/Footer.jsx

```diff
@@ -15,24 +15,34 @@ const Footer = () => {
             <div className="text-sm">Riverside, California 92521</div>
           </div>
 
-          <div className="flex items-center flex-col justify-center  text-white text-5xl border-l-8 border-white pl-4">
+          <div className="flex items-center flex-col justify-center  text-white text-5xl border-l-8 border-white pl-4 ">
             <Link
               href="https://www.instagram.com/ucrprishi/?hl=en"
               target="_blank"
             >
-              <AiFillInstagram style={{ color: "white" }} />
+              <AiFillInstagram
+                className="hover:scale-110"
+                style={{ color: "white" }}
```

Comment:
i didnt see this before but can you remove style={{ color: "white" }} and make it className="bg-rishi-white" instead